src/include/miopen/conv/asm_implicit_gemm.hpp

```diff
@@ -34,6 +34,10 @@
 #include <vector>
 #include <limits>
 
+/// W/A for issue 1979: igemm solver does not support group conv. See:
+/// https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1979
```

Comment:
[Notice] Basically, `WORKAROUND_ISSUE_1979` says everything so the comment is not required.